ui/components/app/snaps/snap-home-page/snap-home-renderer.js

```diff
@@ -64,7 +65,14 @@ export const SnapHomeRenderer = ({ snapId }) => {
   }, [unapprovedTemplatedConfirmations, unapprovedConfirmations, history]);
 
   return (
-    <Box height={BlockSize.Full}>
+    <Box
+      height={BlockSize.Full}
+      width={BlockSize.Full}
+      backgroundColor={BackgroundColor.backgroundAlternative}
+      style={{
+        overflow: 'auto',
```

Comment:
```suggestion
        overflowY: 'auto',
```

Reply:
Same thing as explained here: https://github.com/MetaMask/metamask-extension/pull/26639#discussion_r1745617302

Reply:
Changed to target only Y axis.